### ISSUE TITLE
Test interpolation methods fails under Octave

### ISSUE THREAD

**Issue by hagenw** [sfstoolbox/sfs-matlab#190]
Running the `test_interpolation_methods(1)` in the ´validation/` folder fails for the last two plots.

The first issue is the non-supported legend option `'Location','SW'`.

By removing this option Octave continues and shows the figures, but they contain only the legend and the following warning message is returned:
```
warning: axis: omitting non-positive data in log plot
warning: axis: omitting non-positive data in log plot
warning: axis: omitting non-positive data in log plot
warning: axis: omitting non-positive data in log plot
warning: axis: omitting non-positive data in log plot
warning: axis: omitting non-positive data in log plot
warning: opengl_renderer: data values greater than float capacity.  (1) Scale data, or (2) Use gnuplot
```

**Comment by fietew** [comment]
For the location of the legend
https://github.com/sfstoolbox/sfs-matlab/blob/67d20342d2366c2ea1993d1f23397056f03d7cd6/validation/test_interpolation_methods.m#L250-L251
a compatible value would be ``'southwest'``

The warnings are related to 
https://github.com/sfstoolbox/sfs-matlab/blob/67d20342d2366c2ea1993d1f23397056f03d7cd6/validation/test_interpolation_methods.m#L239-L246
The ```semilogx``` does not allow ``f == 0``.

**Comment by hagenw** [comment]
Thanks.
To get warnings is not a problem, but the plot is also empty. Is this as well related to `f == 0`?

**Comment by fietew** [comment]
I will quickly provide a PR for this.

**Comment by fietew** [comment]
> Thanks.
To get warnings is not a problem, but the plot is also empty. Is this as well related to f == 0?

Yes

**Comment by fietew** [comment]
See #191

**Comment by hagenw** [comment]
Solved by #191
